src/main/java/com/timebook/timebook/models/events/Event.java

```diff
@@ -33,14 +34,16 @@ public Event() {
             String description,
             LocalDateTime startDateTime,
             LocalDateTime endDateTime,
-            int priority) {
+            int priority,
+            Boolean isVisible) {
         this.id = id;
         this.email = email;
         this.title = title;
         this.description = description;
         this.startDateTime = startDateTime;
         this.endDateTime = endDateTime;
         this.priority = priority;
+        this.isVisible = isVisible;
```

Comment:
I just think of a name "isPublic". Would it more accurately describe the attribute?